docs/configuration.md

```diff
@@ -85,6 +92,10 @@ Environment variables are automatically loaded with these mappings:
 - `CODEGATE_CA_KEY`: CA key file name
 - `CODEGATE_SERVER_CERT`: server certificate file name
 - `CODEGATE_SERVER_KEY`: server key file name
+- `CODEGATE_ENABLE_LITELLM`: enable LiteLLM logging
+- `CODEGATE_ENABLE_SQLALCHEMY`: enable SQLAlchemy logging
+- `CODEGATE_ENABLE_UVICORN_ERROR`: enable Uvicorn error logging
+- `CODEGATE_ENABLE_AIOSQLITE`: enable aiosqlite logging
```

Comment:
We probably want to add the word "logging" to these environment variables. Else, it hints at enabling us disabling a whole component